dashboard/app/models/pd/application/application_base.rb

```diff
@@ -81,6 +67,20 @@ class ApplicationBase < ActiveRecord::Base
         'American Indian/Alaska Native',
         OTHER,
         'Prefer not to say'
+      ],
+
+      course_hours_per_year: [
+        'At least 100 course hours',
+        '50 to 99 course hours',
+        'Less than 50 course hours'
+      ],
+
+      terms_per_year: [
+        '1 quarter',
+        '1 trimester',
+        '1 semester',
+        '2 trimesters',
+        'Full year'
```

Comment:
Note: this needs to be updated in the teacher application too (e.g. [here](https://github.com/code-dot-org/code-dot-org/blob/staging-next/dashboard/app/models/pd/application/teacher1819_application.rb#L253)). Luckily this was caught by tests.